R/deployTFModel.R

```diff
@@ -1,45 +0,0 @@
-#' Deploy a TensorFlow saved model
-#'
-#' Deploys a directory containing a Tensorflow saved model file.
-#'
-#' Deploy a single Tensorflow saved model as a bundle. Should be passed a
-#' directory that contains the *saved_model.pb* or *saved_model.pbtxt* file,
-#' as well as any variables and assets necessary to load the model.
-#'
-#' A saved model directory might look like this:
-#'
-#' ```
-#' ./1/
-#' ./1/saved_model.pb or ./1/saved_model.pbtxt
-#' ./1/variables/
-#' ./1/variables/variables.data-00000-of-00001
-#' ./1/variables/variables.index
-#' ```
-#'
-#' For information on creating saved models, see the Keras method
-#' [keras::export_savedmodel.keras.engine.training.Model()] or the TensorFlow
-#' method [tensorflow::export_savedmodel()]. If using the TensorFlow package for
-#' R, the official [TensorFlow guide for saving and restoring models](
-#' <https://www.tensorflow.org/guide/saved_model>)
-#' may be useful.
-#'
-#' TensorFlow, the TensorFlow logo and any related marks are trademarks of
-#' Google Inc., and are not affiliated with Posit Software, PBC.
-#'
-#' @param modelDir Path to the saved model directory. MUST contain
-#'   *saved_model.pb* or *saved_model.pbtxt*
-#' @param ... Additional arguments to [deployApp()].
-#'
-#' @references <https://www.tensorflow.org/guide/saved_model>
-#'
-#'
-#' @details
-#'
-#' @family Deployment functions
-#' @export
```

Comment:
Might be worth keeping this function around with an informative error? Not that important but it is nice to know that it’s definitely gone away, rather than you forgetting the name of the function.

Reply:
@kellobri - What are your thoughts about this? My feeling is that this content type saw so little use that there is almost no risk to fully removing the TF deploy function. I'm happy to retain a skeleton implementation that errs if it'll be helpful.

Reply:
Yes, we saw extremely low usage. Errors are always nice though, I like the idea of the skeleton implementation if that's possible.